python/lsst/ip/isr/crosstalk.py

```diff
@@ -384,7 +456,8 @@ def toTable(self):
 
         if self.interChip:
             interChipTable = Table([{'IC_SOURCE_DET': sourceDet,
-                                     'IC_COEFFS': self.interChip[sourceDet].reshape(self.nAmp*self.nAmp)}
+                                     'IC_COEFFS':
```

Comment:
Is this a line length thing?

Reply:
I changed it a bit now. maybe it was `black`?